R/data.table.R

```diff
@@ -62,7 +62,7 @@ data.table = function(..., keep.rownames=FALSE, check.names=FALSE, key=NULL, str
   if (!is.null(key)) {
     if (!is.character(key)) stop("key argument of data.table() must be character")
     if (length(key)==1L) {
-      key = strsplit(key,split=",")[[1L]]
+      key = trimws(strsplit(key,split=",")[[1L]])
```

Comment:
IIRC `trimws` wasn't in R 3.1.0

Reply:
hmm i may be wrong

https://github.com/wch/r-source/commit/e4f12c04ea2a0c9ff7f3fbfa445ae2ae7300200a

Reply:
yep it's 3.2.0

https://github.com/wch/r-source/blob/trunk/doc/NEWS.Rd#L5382

Reply:
ah right, but we back-ported it 

https://github.com/Rdatatable/data.table/blob/master/R/data.table.R#L1